README.md

```diff
@@ -329,6 +329,27 @@ Normally, setting required DSN information would be enough.
   ```
   - See https://docs.sentry.io/platforms/node/pluggable-integrations/ for more information
 
+### enableTracing
+- Type: `Boolean` or `Object`
+  - Default: `false`
+  - Enables the BrowserTracing integration
+  - A sampling rate can be provided if passing an object (if not provided, the default rate is 1.0):
```

Comment:
I feel that the default sampling rate should be lower (maybe 0.6 or even 0.4?). If users want to be overwhelmed they can raise it to 1. What do you think?

Reply:
My thought was that a new user testing out the feature (just setting the option to `true`) would want their first request captured, with no additional configuration. The Sentry docs do recommend changing this to a lower value, just not sure what a sane value would be. Can always heavily encourage users to configure this key.

Reply:
OK, let's have a warning/advice similar to what Sentry says.